system/settings/settings.xml

```diff
@@ -2253,7 +2235,7 @@
         </setting>
         <setting id="audiooutput.ac3transcode" type="boolean" parent="audiooutput.ac3passthrough" label="667" help="36429">
           <level>2</level>
-          <default>false</default>
+          <default>true</default>
           <dependencies>
```

Comment:
@fritsch yesterday's inclusion was a mistake, so here it is again.

@FernetMenta a ping to see if you are ok with this?
 
The intention of this is to make getting 5.1 audio easier for users of 2.0 channel configurations. At the moment when passthrough is enabled then only the Dolby Digital format is enabled, leaving the others such as DTS disabled. This means in a 2.0 configuration even though a file being played may contain 5.1 the user will only get PCM 2.0 audio for DTS or any of the non-Dolby Digital formats. With Transcoding set to be enabled by default this will mean DTS etc will be output as Dolby Digital 5.1 so the user gets the 5.1 audio with no other user selections other than just enabling passthrough and configuring the output device.

For RPi I've set AC3 transcoding to be off as I noticed the RPi only help statement about CPU resource. @popcornmix  can you confirm this can still cause CPU resource issues if enabled, and does it apply to both RPi and RPi2.